static/js/jira-manual-app-creation.js

```diff
@@ -0,0 +1,28 @@
+$("#jiraManualAppCreation__form").submit((event) => {
+  event.preventDefault();
+
+  // TODO: Form submission
+});
+
+$(".jiraManualAppCreation__formFileInput")
+  .on("dragenter focus", () => {
+    $(".jiraManualAppCreation__formFileDropArea").addClass("active");
+  }).on("dragleave blur drop", () => {
+    $(".jiraManualAppCreation__formFileDropArea").removeClass("active");
+  }).on("change", (event) => {
+    const fileName = $(event.target).val().split('\\').pop();
```

Comment:
This could be problematic on non-windows platform. Anyway there's a cleaner way to do this, use the new file object. You can even get the binary date off that object. 

https://developer.mozilla.org/en-US/docs/Web/API/File

Reply:
Used `event.target.files[0].name`